### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by svekars**
Add a bug report and feature request issue template. 
cc @malfet @ZainRizvi

### CODE REVIEW

**Comment by svekars** [comment]
Preview:
https://github.com/pytorch/tutorials/blob/add-issue-template-2/.github/ISSUE_TEMPLATE/feature-request.yml 
https://github.com/pytorch/tutorials/blob/add-issue-template-2/.github/ISSUE_TEMPLATE/bug-report.yml

**Review by malfet — APPROVED** [review]

**Comment by malfet** [comment]
It would be nice to add a followup to add snippet to a website footer to report issue that would auto-fill the URL

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    label: 🚀 Descirbe the improvement or the new tutorial
+    description: |
+      **Describe the improvement**
+    placeholder: |
+      Explain why this improvement or new tutorial is important. For example, *"This tutorial will help users to better understand feature X of PyTorch."* If there is a tutorial that you propose to replace, add here. If this is related to another GitHub issue, add a link here.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Existing tutorials on this topic
```
Comment (ZainRizvi) [inline .github/ISSUE_TEMPLATE/feature-request.yml:20]: What's the point of linking to existing tutorials?

Putting on my "contributor" hat, if I thought such tutorials already existed I'd prob not have bothered creating this new one

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      The error message you got, with the full traceback.
+      ```
+
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Describe your environment
+    description: |
+      **Describe your environment:**
```
Comment (ZainRizvi) [inline .github/ISSUE_TEMPLATE/bug-report.yml:40]:
```suggestion
      **Describe the environment you encountered the bug in:**
```

**Review by ZainRizvi — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 1 above)
Comment (svekars) [reply .github/ISSUE_TEMPLATE/feature-request.yml:20]: This is if someone is proposing a tutorial on a topic that is already covered but might be outdated, uses old tech and so on. Like a better version of the existing one.

**Pull request merged**